components/SocialLink/SocialLink.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 const SocialLink = (props: any) => {
```

Comment:
I missed this ... no unnecessary `any` types. Please give this component proper types.
Also, write unit tests for it... you can look for any files under a `__tests__` folder, or with `.spec.tsx` extension.